ref-img.sh

```diff
@@ -0,0 +1,9 @@
+if [ "$TEST_SUITE" = "css:test --set set1" ]; then
+    wget $SET1;
```

Comment:
Where does $SET1 get set? I thought it would be in the Travis config but didn't see it.